ArduPlane/control_modes.cpp

```diff
@@ -161,9 +161,19 @@ void Plane::reset_control_switch()
 void Plane::autotune_start(void)
 {
     gcs().send_text(MAV_SEVERITY_INFO, "Started autotune");
```

Comment:
I wonder if we would print a warning if axis is 0. In that case it would print "Started autotune" but not do anything.